shared-libs/user-management/src/users.js

```diff
@@ -335,21 +351,21 @@ const hasParent = (facility, id) => {
 const mapUsers = (users, settings, facilities) => {
   users = users || [];
   return users
-    .filter(user => user.id.indexOf(USER_PREFIX) === 0)
+    .filter(user => user._id.indexOf(USER_PREFIX) === 0)
     .map(user => {
-      const setting = getDoc(user.id, settings) || {};
+      const setting = getDoc(user._id, settings) || {};
       return {
-        id: user.id,
-        rev: user.doc._rev,
-        username: user.doc.name,
+        id: user._id,
+        rev: user._rev,
+        username: user.name,
         fullname: setting.fullname,
         email: setting.email,
         phone: setting.phone,
-        place: getDoc(user.doc.facility_id, facilities),
-        roles: user.doc.roles,
+        place: getDoc(user.facility_id, facilities),
```

Comment:
IMHO we really need to switch this to `settings.facility_id` or else change to query the `_users` table by `facility_id`.  Using both like this is going to make for some debugging hell later on....